app/settings_test.go

```diff
@@ -76,16 +76,68 @@ func TestOverrideTheme_IgnoresSettingsChange(t *testing.T) {
 	}
 	set.setupTheme()
 	assert.Equal(t, theme.LightTheme(), set.Theme())
+
```

Comment:
I have edited this test, by removing the behavior that ignores forever the changes in the application once FYNE_THEME env var is set at least once, is that ok?

Reply:
FYNE_THEME is an override, once set it should not be changed

Reply:
I don't think we had considered the env changing during runtime. That could go either way.
What I meant was that if env is set nothing else should influence the setting